doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2873,6 +2880,18 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       hypotheses, which without the flag it may break.
       default.
 
+   .. exn:: Cannot find any non-recursive equality over :n:`@ident`.
+      :undocumented:
+
+   .. exn:: Section variable :n:`@ident` occurs implicitly in global
+            declaration :n:`@qualid` present in hypothesis :n:`@ident`.
+
+            Section variable :n:`@ident` occurs implicitly in global
+            declaration :n:`@qualid` present in the conclusion.
```

Comment:
If your goal is to document two variations of the error message, then you should do:
```suggestion
   .. exn:: Section variable :n:`@ident` occurs implicitly in global declaration :n:`@qualid` present in hypothesis :n:`@ident`.
            Section variable :n:`@ident` occurs implicitly in global declaration :n:`@qualid` present in the conclusion.
```

or:
```suggestion
   .. exn:: Section variable :n:`@ident` occurs implicitly in global \
            declaration :n:`@qualid` present in hypothesis :n:`@ident`.
            Section variable :n:`@ident` occurs implicitly in global \
            declaration :n:`@qualid` present in the conclusion.
```

(I discovered that the second one was possible only recently.)